tasks/common.yml

```diff
@@ -21,7 +21,7 @@
     notify:
       - restart sensu-server service
       - restart sensu-api service
-      - restart sensu-client service
```

Comment:
`sensu-client` service may not exist in sensu master node for enterprise users (`sensu-enterprise` only comprises of `sensu-server` and `sensu-api` [Reference)](https://sensuapp.org/docs/0.28/platforms/sensu-on-rhel-centos.html#sensu-enterprise). 

Restart a non-existing service may fail the playbook. Besides, I don't know if it is a common installation what sensu master will monitor itself as a client.